client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -526,29 +552,14 @@ export class PlanFeatures2023Grid extends Component {
 	}
 }
 
-PlanFeatures2023Grid.propTypes = {
-	isInSignup: PropTypes.bool,
-	onUpgradeClick: PropTypes.func,
-	// either you specify the plans prop or isPlaceholder prop
-	plans: PropTypes.array,
-	visiblePlans: PropTypes.array,
-	planProperties: PropTypes.array,
-	flowName: PropTypes.string,
-};
-
-PlanFeatures2023Grid.defaultProps = {
-	isInSignup: true,
-	onUpgradeClick: noop,
-};
-
 /* eslint-disable wpcalypso/redux-no-bound-selectors */
 export default connect(
-	( state, ownProps ) => {
+	( state, ownProps: any ) => {
 		const { placeholder, plans, isLandingPage, visiblePlans } = ownProps;
 
-		let planProperties = plans.map( ( plan ) => {
+		let planProperties: PlanProperties[] = plans.map( ( plan: any ) => {
 			let isPlaceholder = false;
-			const planConstantObj = applyTestFiltersToPlansList( plan, undefined );
+			const planConstantObj: any = applyTestFiltersToPlansList( plan, undefined );
```

Comment:
You shouldn't need to type this at all since `applyTestFiltersToPlansList` has a return type already. (In fact, adding the `any` type here might hide bugs.)

```suggestion
			const planConstantObj = applyTestFiltersToPlansList( plan, undefined );
```